homeassistant/components/device_tracker/locative.py

```diff
@@ -76,21 +78,25 @@ def _handle(self, data):
         device = data['device'].replace('-', '')
         location_name = data['id'].lower()
         direction = data['trigger']
+        gps_location = (data[ATTR_LATITUDE], data[ATTR_LONGITUDE])
 
         if direction == 'enter':
             yield from self.hass.loop.run_in_executor(
                 None, partial(self.see, dev_id=device,
-                              location_name=location_name))
+                              location_name=location_name,
+                              gps=(data[ATTR_LATITUDE], data[ATTR_LONGITUDE])))
```

Comment:
Why not use the gps_location variable? Otherwise you can remove the gps_location variable.